src/sentry/api/endpoints/organization_details.py

```diff
@@ -261,6 +262,20 @@ def validate_projectRateLimit(self, value):
             )
         return value
 
+    def validate_providerName(self, value):
+        from sentry.auth import manager
+
+        has_api_auth_provider = features.has(
+            "organizations:api-auth-provider",
+            self.context["organization"],
+            actor=self.context["request"].user,
+        )
+        if not has_api_auth_provider:
+            return
```

Comment:
It might make more sense to raise an error, so people on self-hosted would know they need to enable the feature flag (compared to a silent failure)